README.md

```diff
@@ -76,12 +83,21 @@ async function connectToWhatsApp () {
             console.log('opened connection')
         }
     })
+
+    // you need to save your credentials for when it restart the connection.
+    sock.ev.on('creds.update', async () => {
+        await saveCreds()
+    });
+
+    // this will send a message to yourself 5 seconds after you log in.
     sock.ev.on('messages.upsert', m => {
         console.log(JSON.stringify(m, undefined, 2))
 
         console.log('replying to', m.messages[0].key.remoteJid)
-        await sock.sendMessage(m.messages[0].key.remoteJid!, { text: 'Hello there!' })
+        setTimeout(() => sock.sendMessage(m.messages[0].key.remoteJid!, { text: 'Hello there!' }), 5000);
```

Comment:
no need, self implementation

Reply:
> We already possess an example file.. please ensure the readme remains concise and straightforward

this old code for this one is kinda of broken, if you copy and paste it does not work, that's why I did this PR, for new joiners to simply copy and paste and it works.

it was also outdated as it didn't have the "auth" prop on config, which is now required, probably by the time it was written it wasn't required.

Reply:
> no need, self implementation

the first time it did work I didn't realise but it simply started a loop of sending messages, got like 100 messages within seconds, and even got a bit scared of getting my account blocked or something.

the listener of "messages.upsert" is basically sending a new message that triggers this one again, which ends up sending too much, that's why the timeout, to do it every 5 seconds istead.